src/app/item-page/simple/item-page.component.scss

```diff
@@ -4,3 +4,7 @@
     max-width: none;
   }
 }
+.request-a-copy-access-icon {
+  margin-right: 4px;
+  color: #26a269;
```

Comment:
This colour is defined in two places.

Could you use a Bootstrap variable like `--bs-success` for it, or create a new `--ds-` variable in `_custom_variables.scss` to make sure it's easy to theme?